### PR TITLE
CTOR-1062 : Refresh cloud-aws-sqs

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-sqs

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md [hunk 1]
```diff
@@ -53,20 +69,15 @@ https://docs.aws.amazon.com/fr_fr/AWSSimpleQueueService/latest/SQSDeveloperGuide
...
-Ce compte doit bénéficier des privilèges suivants :
-
-| AWS Privilege                  | Description                                          |
-|:-------------------------------|:-----------------------------------------------------|
-| sqs:ListQueues                 | Returns a list of your queues in the current region. |
-| cloudwatch:listMetrics         | List all metrics from Cloudwatch AWS/VPN namespace   |
-| cloudwatch:getMetricStatistics | Get metrics values from Cloudwatch AWS/VPN namespace |
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
+* cloudwatch:getMetricStatistics
+* sqs:ListQueues
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md:74]:
```suggestion
* cloudwatch:getMetricStatistics
* cloudwatch:listMetrics
* sqs:ListQueues
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-sqs.md [hunk 2]
```diff
@@ -54,19 +68,15 @@ https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-m
...
-Configure a service account (*access/secret keys* combo) for which the following privileges have to be granted:
-
-| AWS Privilege                  | Description                                          |
-|:-------------------------------|:-----------------------------------------------------|
-| sqs:ListQueues                 | Returns a list of your queues in the current region. |
-| cloudwatch:listMetrics         | List all metrics from Cloudwatch AWS/VPN namespace   |
-| cloudwatch:getMetricStatistics | Get metrics values from Cloudwatch AWS/VPN namespace |
+Configure a service account (access/secret key combo) for which the following privileges have to be granted:
+* cloudwatch:getMetricStatistics
+* sqs:ListQueues
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-sqs.md:73]:
```suggestion
* cloudwatch:getMetricStatistics
* cloudwatch:listMetrics
* sqs:ListQueues
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md [hunk 3]
```diff
@@ -5,46 +5,62 @@ title: Amazon SQS
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md [hunk 4]
```diff
@@ -5,46 +5,62 @@ title: Amazon SQS
...
 
 ### Règles de découverte
 
-Ce pack propose une règle de découverte d'hôtes permettant de découvrir automatiquement des ressources **AWS SQS** : 
+#### Découverte d'hôtes
 
-![image](../../../assets/integrations/plugin-packs/procedures/cloud-aws-sqs-provider.png)
+| Nom de la règle        | Description                 |
+|:-----------------------|:----------------------------|
+| Amazon Web Service SQS | Découvre les Topics AWS SNS |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md:43]: topics sns??

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md [hunk 5]
```diff
@@ -53,20 +69,16 @@ https://docs.aws.amazon.com/fr_fr/AWSSimpleQueueService/latest/SQSDeveloperGuide
...
 
-Un compte de service (paire d'identifiants *access/secret keys*) est nécessaire afin de pouvoir superviser les resources **Amazon SQS**.
-Ce compte doit bénéficier des privilèges suivants :
-
-| AWS Privilege                  | Description                                          |
-|:-------------------------------|:-----------------------------------------------------|
-| sqs:ListQueues                 | Returns a list of your queues in the current region. |
-| cloudwatch:listMetrics         | List all metrics from Cloudwatch AWS/VPN namespace   |
-| cloudwatch:getMetricStatistics | Get metrics values from Cloudwatch AWS/VPN namespace |
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-sqs.md:72]:
```suggestion
Configurez un compte de service (via une combinaison d'access key et de secret key) et affectez-lui les privilèges suivants :
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-sqs.md [hunk 6]
```diff
@@ -5,47 +5,61 @@ title: Amazon SQS
...
 
 ### Discovery rules
 
-The pack provides a discovery rule to automatically discover SQS resources:
+#### Host discovery
 
-![image](../../../assets/integrations/plugin-packs/procedures/cloud-aws-sqs-provider.png)
+| Rule name              | Description             |
+|:-----------------------|:------------------------|
+| Amazon Web Service SQS | Discover AWS SNS Topics |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-sqs.md:42]: SNS topics??

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/cloud-aws-sqs.md (hunk 6 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-sqs.md:42]:
```suggestion
| Amazon Web Service SQS | Discover AWS SQS Topics |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**